web/src/components/layout/Header.tsx

```diff
@@ -32,11 +33,82 @@ import {
   ToolbarContent,
   ToolbarGroup,
   ToolbarItem,
+  Dropdown,
+  MenuToggleElement,
+  MenuToggle,
+  DropdownList,
+  DropdownItem,
+  Divider,
 } from "@patternfly/react-core";
 import { Icon } from "~/components/layout";
-import { InstallerOptions } from "~/components/core";
 import { useProduct } from "~/queries/software";
 import { _ } from "~/i18n";
+import { InstallationPhase } from "~/types/status";
+import { useInstallerStatus } from "~/queries/status";
+import { InstallerOptions } from "../core";
+import { useLocation } from "react-router-dom";
+import { PATHS } from "~/router";
+
+export type HeaderProps = {
+  /** Whether the application sidebar should be mounted or not */
+  showSidebarToggle?: boolean;
+  /** Whether the selected product name should be shown */
+  showProductName?: boolean;
+  /** Whether the installer options link should be mounted */
+  showInstallerOptions?: boolean;
+  /** The background color for the top bar */
+  background?: MastheadProps["backgroundColor"];
+};
+
+const OptionsDropdown = ({ showInstallerOptions }) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [isInstallerOptionsOpen, setIsInstallerOptionsOpen] = useState(false);
+  const toggle = () => setIsOpen(!isOpen);
+  const toggleInstallerOptions = () => setIsInstallerOptionsOpen(!isInstallerOptionsOpen);
+
+  return (
+    <>
+      <Dropdown
+        popperProps={{ position: "right", appendTo: () => document.body }}
+        isOpen={isOpen}
+        onOpenChange={toggle}
+        onSelect={toggle}
+        onActionClick={toggle}
+        toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
+          <MenuToggle
+            ref={toggleRef}
+            onClick={toggle}
+            aria-label={_("Options toggle")}
```

Comment:
:sos: Help needed to look for a better label for this "application options menu/dropdown". Please, note that it is not "Installer options", but contains an entry for it. What is more, known "Installer options" are actually a "Installer Localization options", but both terms are kind of... "confusing?"